apps/src/weblab/brambleHost.js

```diff
@@ -313,26 +313,25 @@ function uploadAllFilesFromBramble(callback) {
       if (i < entries.length) {
         const entry = entries[i];
         getFileData(entry.path, (err, fileData) => {
-          if (err) {
-            callback();
-          } else {
+          callback(err);
```

Comment:
Just to confirm my understanding here - it is okay to call the callback with (err) even if !err because the callback handles that?

Reply:
Added a comment to help clear up future confusion